src/include/network/lan/address.rb

```diff
@@ -1593,14 +1546,72 @@ def AddressDialog
 
   private
 
-    def get_selected_bridges
+    # Checks if the current LanItem is a bridge or an enslaved interface part
+    # of a bridge. In case that some configuration is not correct then adapt it
+    # if the user accepts.
+    def check_bridge_ports_config
+      return false unless LanItems.type == "br"
```

Comment:
This was exactly the reason I proposed the new method for adapt the config, because the other ones depends on the NetworkInterfaces instance, and it also changed the Current to the last port edited what could produce some unexpected issues.